mcx/sample.py

```diff
@@ -390,7 +390,7 @@ def sample_scan(
         f"Collecting {num_samples:,} samples across {num_chains:,} chains",
         refresh=False,
     )
-    print_rate = int(num_samples / 10)
+    print_rate = int(num_samples / 20)
```

Comment:
The progress bar now progresses a bit more smoothly which is especially nice for long runs. I did a quick test and there doesn't seem to be a performance penalty

Reply:
Ok!